### DISCUSSION TITLE
Document_Root is not defined

### DISCUSSION THREAD

**Discussion by seri3s** [zim-desktop-wiki/zim-desktop-wiki#2395] [General]
Hi,

I am a newvie to ZIM.

I installed ZIM Desktop wiki 0.73.5 on my Raspi-64bit PI-OS.

It worked well but when I open Document_Root pop-up window come to say "Document_Root is not defined ...".

I guess that Document_Root seems like ~/NoteBoos/.
I appled the code to  ~/.config/preferences below
  [General]
document_dir = ~/NoteBooks     <<< inserted
--
But it says the same error.

Does someone help me how to define Document Root of ZIM?

Thanks and Regards,

**Comment by introt** [comment]
Your setup seems to have some errors. `document_root` (not `document_dir`) should be set in `notebook.zim`, under `[Notebook]`. Do not add spaces around the `=`.
